templates/zerver/integrations/errbot.md

```diff
@@ -25,23 +27,42 @@ Run your favorite chatbot in Zulip!
         BOT_LOG_FILE = r'<path/to/your/errbot/logfile.log>'
         BOT_LOG_LEVEL = logging.INFO
 
-        BOT_IDENTITY = {  # Fill this with the corresponding values in your bot's `.zuliprc`
-          'email': '<err-bot@your.zulip.server>',
+        BOT_IDENTITY = {
+          'email': '<your-bot-name@yourZulipDomain.zulipchat.com>',
           'key': '<abcdefghijklmnopqrstuvwxyz123456>',
-          'site': '<http://your.zulip.server>'
+          'site': '<https://yourZulipDomain.zulipchat.com>'
         }
-        BOT_ADMINS = ('<your@email.address',)
+        BOT_ADMINS = ('<your@email.address>',)
         CHATROOM_PRESENCE = ()
-        BOT_PREFIX = '<@**err-bot**>'  # Providing errbot's full name in Zulip lets it respond to @-mentions.
+        BOT_PREFIX = '@**<your-bot-name>**'
+
+    !!! tip ""
```

Comment:
Newly added tip.

We have several other integration docs that would benefit from directly linking to the `#settings/your-bots` page. So, finalizing the wording around this would be helpful.
I considered adding a macro for this, but it's not worth adding a new instruction bullet just for this, it might be better to use it differently for each integration. While we need an explicit tip for this doc, there are several docs where we could just hyperlink "your bot's info" that is already part of an instruction, without creating a separate line.

Also, we can't use the `[][]` reference link for these, the `zulip_url` variable doesn't work with that.

Reply:
That link won't work for the general `zulip.com/integrations` case, so I think we should just write where in the web app to get the bot's information.